src/components/App.tsx

```diff
@@ -1,10 +1,18 @@
-import React from "react";
+// Visuele onderdelen
 import "../App.css";
-import Header from "./general/header";
 import { Container, Card, CardContent } from "@mui/material";
+
+// React
+import React from "react";
```

Comment:
Het lijkt me goed om alles wat react is bovenaan te zetten. Anders krijgen we mogelijk problemen met dependencies

Reply:
Aangepast!